refactor(app): deck cal button says recalibrate

sfoster1 opened the pull request:
When we have a deck calibration already, the deck cal button should say
recalibrate rather than calibrate.

Also slightly change the migrated content note.

sfoster1 committed:
refactor(app): deck cal button says recalibrate

When we have a deck calibration already, the deck cal button should say
recalibrate rather than calibrate.

Also slightly change the migrated content note.

sfoster1 committed:
fixup: format

sfoster1 committed:
sometimes i DO want to coerce, OKAY

ahiuchingau reviewed (changes requested):
code lgtm!

Just one small nitpick - can we align the calibrate deck button to the center of the Calibrate Deck box so it's not at the top?
<img width="639" alt="Screen Shot 2020-11-05 at 2 26 46 PM" src="https://user-images.githubusercontent.com/20424172/98287167-fb6eaf80-1f72-11eb-8dac-3f54b2ccf26c.png">

Laura-Danielle reviewed (approved):
lgtm

sfoster1 committed:
push titled controls to align-center

This makes the deck calibration button centered, and doesn't really
affect anything else because there's not enough content there

ahiuchingau reviewed (approved):
Beautiful ✨
